vagrant/base.sh

```diff
@@ -18,7 +18,7 @@ set -ex
 
 # The version of Kibosh to use for testing.
 # If you update this, also update tests/docker/Dockerfile
-export KIBOSH_VERSION=d85ac3ec44be0700efe605c16289fd901cfdaa13
+export KIBOSH_VERSION=8841dd392e6fbf02986e2fb1f1ebf04df344b65a
```

Comment:
Just linking this here for posterity https://github.com/confluentinc/kibosh/commit/8841dd392e6fbf02986e2fb1f1ebf04df344b65a